### PR TITLE
Add remove-decorators extension

### PR DESCRIPTION
**Pull request by Pawel-Kica**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Pawel-Kica** [comment]
Simple extension to replace 
```@ObjectType()
@Entity(tableNames.users)
export class User {
    @Field()
    @PrimaryGeneratedColumn('uuid')
    user_id: string;

    @Field()
    @Column({ unique: true })
    username: string;
}
```
with this
```
export class User {
    user_id: string;

    username: string;
}
```
useful for NestJS types sharing

**Comment by Pawel-Kica** [comment]
based on json-format existing extension https://github.com/raycast/extensions/tree/59087c124218d0aa70034caa193aaa6d15a905a7/extensions/json-format/

File: extensions/remove-decorators/src/removeDecoratorsClipboard.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Clipboard } from "@raycast/api";
+import removeDecorators from "./utils/removeDecorators";
+
+export default async () => {
+  await removeDecorators((await Clipboard.readText()) || "");
```
Comment (sxn) [inline extensions/remove-decorators/src/removeDecoratorsClipboard.tsx:5]: you could also use [getSelectedText](https://developers.raycast.com/api-reference/environment#getselectedtext) – should improve UX 😃

File: extensions/remove-decorators/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Everything
```
Comment (sxn) [inline extensions/remove-decorators/README.md:1]: mind adding something to the readme?

File: extensions/remove-decorators/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Everything Changelog
+
+## [Initial Version] - 2022-10-23
```
Comment (sxn) [inline extensions/remove-decorators/CHANGELOG.md:3]: this date is about 6 months in the past 😅

File: extensions/remove-decorators/src/removeDecoratorsClipboard.tsx (hunk 1 above)
Comment (sxn) [inline extensions/remove-decorators/src/removeDecoratorsClipboard.tsx:5]: you could also use [getSelectedText](https://developers.raycast.com/api-reference/environment#getselectedtext) – should improve UX 😃

**Review by sxn — CHANGES_REQUESTED** [review]
hey @Pawel-Kica, thanks for the PR! I left a couple of suggestions for improvements, let me know when you've had time to address them 😺

**Comment by Pawel-Kica** [comment]
Thank you, will review it!!!

**Pull request closed without merging**